webapp/src/modules/wallet/sagas.js

```diff
@@ -55,6 +56,10 @@ function* handleConnectWalletSuccess(action) {
     }
   }
 
+  if (isFeatureEnabled('BIDS')) {
+    authorization.allowances['ERC721Bid'] = ['MANAToken']
```

Comment:
You can use a dot notation here if you want
```suggestion
    authorization.allowances.ERC721Bid = ['MANAToken']
```